app/controllers/repositories_controller.rb

```diff
@@ -1,5 +1,5 @@
 class RepositoriesController < ApplicationController
-  before_action :set_repository, only: [:show, :toggle_star]
+  before_action :set_repository, only: [:show, :toggle_star, :number_of_comments]
```

Comment:
which controller is defining this `:number_of_comments` filter. Because I can't see it.